bin/release-package.php

```diff
@@ -132,11 +171,21 @@ function execute( $command, $error = '', $cleanup_repo = false, $cleanup_remotes
  * Cleanup repository and remotes.
  * Should be called at any error that changes the repo, or at success at the end.
  *
- * @param bool $cleanup_repo    Whether to cleaup repo on error.
- * @param bool $cleanup_remotes Whether to cleanup remotes on error.
+ * @param bool   $cleanup_repo    Whether to cleaup repo on error.
+ * @param bool   $cleanup_remotes Whether to cleanup remotes on error.
+ * @param string $tag_version     Optional version tag that needs to be deleted locally.
  */
-function cleanup( $cleanup_repo = false, $cleanup_remotes = false ) {
+function cleanup( $cleanup_repo = false, $cleanup_remotes = false, $tag_version = '' ) {
 	if ( $cleanup_repo ) {
+		// Delete the local tag we created and pushed to the package remote.
+		if ( ! empty( $tag_version ) ) {
+			$command = sprintf(
+				'git tag --delete v%1$s',
```

Comment:
Why do we need to remove package tags?

Reply:
I'm guessing so that they don't clutter the original remote. They will be considered as potential release drafts by GitHub, so we're making sure we're only pushing them to the package remote.